src/auto_scheduler/loop_state.cc

```diff
@@ -239,6 +239,27 @@ Iterator State::vectorize(int stage_id, const Iterator& it) {
   return step->ApplyToState(this);
 }
 
+Array<Iterator> State::follow_split(int stage_id, const Iterator& it, int src_step_id,
```

Comment:
Put these two functions right behind the State::split().
And be care of the order in **other positions**, they should all be put right behind the split step. (e.g. `TVM_REGISTER_GLOBAL` in this file, `follow_split/follow_fused_split` in `loop_state.h`, classes in `transform_steps.h`)